benchmarking_workflows/README.md

```diff
@@ -23,25 +23,25 @@ This is where the summary workflows for APAeval live. The dedicated code for eac
   - [3. Containers & images](#3-containers--images)
 - [Origin](#origin)
 ## Overview
-![apaeval-swfs][apaeval-swfs]
+![apaeval-bwfs][apaeval-bwfs]
 
-APAeval consists of a number of ***benchmarking events*** to evaluate the performance of different tasks that the methods of interest (=***participants***) might be able to perform: **poly(A) site identification, absolute quantification, relative quantification and assessment of their differential usage**. A method can participate in one or several events, depending on its functions.   
+APAeval consists of a number of ***benchmarking events*** to evaluate the performance of different tasks that the methods of interest (=[***participants***][participants]) might be able to perform: **poly(A) site identification, absolute quantification, relative quantification and assessment of their differential usage** (not implemented yet). A method can participate in one or several events, depending on its functions.   
 
-Within a benchmarking event, one or more ***challenges*** will be performed. A **challenge is primarily defined by the ground truth dataset used for performance assessment**. A challenge is evaluated within a ***summary workflow***, which can be run with either docker or singularity, locally or on an HPC infrastructure (currently a profile for Slurm is included). The summary workflow will **compute all metrics relevant for the benchmarking event**. A list of challenge IDs and input files (= output files of one participant for all specified challenges) is passed to the workflow.    
+Within a benchmarking event, one or more ***challenges*** will be performed. A **challenge is primarily defined by the ground truth dataset used for performance assessment** (see [overview of samples][challenges]). A challenge is evaluated within a ***benchmarking workflow***, which can be run with either docker or singularity, locally or on an HPC infrastructure (currently a profile for Slurm is included). The benchmarking workflow will **compute all metrics relevant for the benchmarking event**. A list of challenge IDs and input files (= output files of one participant for all specified challenges) is passed to the workflow.    
```

Comment:
RE HPC job submission, could specify that both Snakemake and nextflow have default profiles available to submit jobs to common clusters?

e.g. https://www.nextflow.io/docs/latest/executor.html
https://github.com/Snakemake-Profiles